### ISSUE TITLE
Add aria-current to SelectItem

### ISSUE THREAD

**Issue by csmatt** [radix-ui/primitives#2828]
### Overview

Currently, the selected `SelectItem` does not have the `aria-current` attribute set to true. This attribute lets screen readers know which item is selected so it can relay that information to the user. NVDA is a screen reader that is free and will help a tester hear the difference. I understand that the spec should not require this, but the screen readers like JAWS and NVDA seem to. Another option, if possible, would be to allow access to the context form outside the component so that I can wrap `SelectItem` and pass it the attribute myself.

### Who does this impact? Who is this for?

Users of my application who are navigating with a screen reader like JAWS or NVDA.
